src/pages/Run/components/RunConfigurationPanel.tsx

```diff
@@ -0,0 +1,29 @@
+import { ChevronRight } from 'react-feather'
+
+import { RunConfiguration } from '@/benchttp'
+import { Button, Typography } from '@/components'
+import { inputConfig } from '@/examples/inputConfig'
+
+interface Props {
+  start: (config: RunConfiguration) => void
+  redirectOnStart: () => void
+}
+
+export const RunConfigurationPanel: React.FC<Props> = ({
```

Comment:
Side note, I also prefer having `Props` declared in the same file as the component, it makes the reading of the component's signature much easier.

@alex-mngt any thoughts on that?

Reply:
I do agree, I think that interfaces and types are best located right next to their direct consumers, ie inside the module defining the component here.

We may drop the file `core/foo.typing.ts`. Only issue I foresee is when helpers in the context need the types (see `getClassNames` using the `Props`' fields).

I guess we need to discuss it soon and define a guideline.

Reply:
> Only issue I foresee is when helpers in the context need the types (see `getClassNames` using the `Props`' fields).

The proper way would be to define common types in `core` and let `Props` and helpers pick whatever they need from them.
For `getClassNames` specifically, we could also declare it in the same file as the component because it's not meant to be reused anywhere and it reduces the "import scope" pollution (and potential errors)
<img width="437" alt="Capture d’écran 2022-09-10 à 13 48 09" src="https://user-images.githubusercontent.com/60700958/189481923-f14f7c73-8dba-41fe-ab61-0519e62d8e51.png">

That say we kinda already agreed on the existing convention so I wouldn't change it unless we really need to.